.github/workflows/pre-submit.e2e.generic.default.yml

```diff
@@ -36,6 +36,7 @@ jobs:
       # echo "2e0390eb024a52963db7b95e84a9c2b12c004054a7bad9a97ec0c7c89d4681d2    binary-name" | base64 -w0
       base64-subjects: "MmUwMzkwZWIwMjRhNTI5NjNkYjdiOTVlODRhOWMyYjEyYzAwNDA1NGE3YmFkOWE5N2VjMGM3Yzg5ZDQ2ODFkMiAgICBiaW5hcnktbmFtZQo="
       compile-generator: true
+      provenance-name: build.intoto.jsonl
```

Comment:
ah good catch! Were these effectively colluding with each other?

Reply:
Yes, which upload-artifact@v4 doesn't allow anymore.

Reply:
Before, I hadn't realized that this particular test was also failing.